_posts/2019-12-28-RLadiesGNV2019.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: posts
+title: "R-Ladies Gainesville: 2019 is a wrap!"
+date: 2019-12-28
+excerpt: "R-Ladies 2019 recap"
```

Comment:
The excerpt is used on the [posts page](https://www.uf-carpentries.org/posts/), so it should be a sentence or two that describes the post.